core/Cargo.toml

```diff
@@ -414,23 +413,18 @@ tokio = { version = "1.27", features = ["time"] }
 uuid = { version = "1.14", features = ["serde", "v4", "js"] }
 
 [dev-dependencies]
-criterion = { version = "0.5", features = ["async", "async_tokio"] }
 divan = { version = "0.1" }
 dotenvy = "0.15"
 fastrace = { version = "0.7", features = ["enable"] }
-fastrace-jaeger = "0.7"
 libtest-mimic = "0.8"
 opentelemetry = { version = "0.29.0", default-features = false, features = [
   "trace",
 ] }
-opentelemetry-otlp = { version = "0.29.0", features = ["grpc-tonic"] }
-opentelemetry_sdk = { version = "0.29.0", features = ["rt-tokio"] }
```

Comment:
maybe this is needed for tracing

Reply:
no releated code use it, if u need, u just import it in high level crate

Reply:
the doctest is marked as `no_run`, but run in ci🤔

Reply:
make sense

Reply:
I knew, no_run mark just let ci not run the code, but compile it. I will fix it